### PR TITLE
Add tests for Openpulse frontend common module

### PR DESCRIPTION
**Pull request by itoko**
### Summary
Add tests for Openpulse frontend common module

### Details and comments
- fix a bug in acquire instruction
- change constructor interface of `Interval`
- add tests for timeslots

### CODE REVIEW

File: qiskit/pulse/commands/acquire.py [hunk 1]
```diff
@@ -106,8 +106,9 @@ def __init__(self,
             reg_slots = []
 
         # TODO: more precise time-slots
```
Comment (taalexander) [inline qiskit/pulse/commands/acquire.py]: Is this todo still required?

File: test/python/pulse/test_timeslots.py [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+class TestTimeslotCollection(QiskitTestCase):
+    """TimeslotCollection tests."""
+
+    def test_can_create_with_valid_timeslots(self):
+        """Test valid time-slot collection creation without error.
+        """
+        slots = [Timeslot(Interval(1, 3), AcquireChannel(0)),
+                 Timeslot(Interval(3, 5), AcquireChannel(0))]
+        _ = TimeslotCollection(slots)
```
Comment (taalexander) [inline test/python/pulse/test_timeslots.py]: there is no need to capture the return?

File: qiskit/pulse/commands/acquire.py (hunk 1 above)
Comment (itoko) [reply qiskit/pulse/commands/acquire.py]: This TODO is for the future when we have `acquisition_latency`.

**Comment by itoko** [comment]
I have no plan to move the contents of common to top-level pulse modules. But I think we have no problem to do so.

**Pull request merged**